e2e/benchmarks/browserstack-benchmark/package.json

```diff
@@ -29,16 +29,17 @@
   "scripts": {
     "test": "karma start",
     "test-node": "jasmine app_node_test.js",
-    "build-deps": "yarn build-core && yarn build-backend-cpu && yarn build-backend-wasm && yarn build-backend-webgl && yarn build-converter && yarn build-layers && yarn build-data && yarn build-tfjs",
-    "build-tfjs": "cd ../../../tfjs && yarn && yarn build && yarn build-npm",
+    "build-all-deps": "yarn build-link-package && yarn build-tfjs",
+    "build-tfjs": "cd ../../../tfjs && yarn && yarn build-npm",
+    "build-link-package": "cd ../../../link-package && yarn build",
     "build-backend-cpu": "cd ../../../tfjs-backend-cpu && yarn && yarn build-npm",
     "build-backend-wasm": "cd ../../../tfjs-backend-wasm && yarn && yarn build && yarn build-npm",
     "build-backend-webgl": "cd ../../../tfjs-backend-webgl && yarn && yarn build && yarn build-npm",
     "build-converter": "cd ../../../tfjs-converter && yarn && yarn build && yarn build-npm",
     "build-core": "cd ../../../tfjs-core && yarn && yarn build && yarn build-npm",
     "build-data": "cd ../../../tfjs-data && yarn && yarn build",
     "build-layers": "cd ../../../tfjs-layers && yarn && yarn build && yarn build-npm",
```

Comment:
You can remove the `build-[some-package]` scripts that you replaced with `build-link-package`. This includes all the Bazel packages, although you'll have to leave the non-bazel packages' scripts here.

Reply:
Thank you Matt!

I just updated the scripts here, as:
```
    "build-all-link-packages": "cd ../../../link-package && yarn build",
    "build-individual-link-package": "cd ../../../link-package && yarn build-deps-for",
```